steepshot_io/graph/views.py

```diff
@@ -39,110 +49,86 @@ def group_steem_golos(self, list_steem, list_golos):
             idx += 1
         return res
 
-    def get_posts_count_monthly(self, platform=None):
-
+    def _get_data(self, platform=None, reverse=True, data_x=None, data_y=None, name_url=None):
         if platform == 'steem':
-            res = requests.get(settings.REQUESTS_URL_STEEM['posts_count_monthly']).json()
-            res.reverse()
+            try:
+                res = requests.get(settings.REQUESTS_URL_STEEM[name_url]).json()
+            except Exception as e:
```

Comment:
Please handle JSONDecodeError and other requests error, like:

``` python
try:
    # some code
except JSONDecodeError as e:
    logger.error('Failed to parse json')
except ( ConnectionError, HTTPError) as e:
    logger.error('Failed to connect to....')
except Exception as e:
    logger.error('Other errors')
```
See the following docs for requests module: http://docs.python-requests.org/en/master/user/quickstart/#errors-and-exceptions